src/userscores/qml/MuseScore/UserScores/internal/TitleListView.qml

```diff
@@ -66,8 +79,15 @@ Item {
         currentIndex: 0
 
         delegate: ListItemBlank {
+            id: item
```

Comment:
It seems that you don't need explicitly set id for delegates

Reply:
This is for `onNavigationActived: item.clicked()`